docs: updates to doc based on reviews (#5)
IBM/infra-to-app-with-landing-zone: docs/part2/30-add-apache.md

![Diagram of the flow through the jump box to the workload VSIs](../images/part-2/media/image21.png)

## Execute the Apache deployment logic 

> You can find the code for this step in the [app-install](https://github.com/IBM/infra-to-app-with-landing-zone/tree/main/app-install) directory. The directory contains the important files:

Reviewer: ```suggestion
You can find the code for this step in the [app-install](https://github.com/IBM/infra-to-app-with-landing-zone/tree/main/app-install) directory. The directory contains the following important files:
```

   
    Notice the following settings in the `main.tf` file:

    - In the connection block, the `bastion_host` is set to the management server floating IP address that you will use as a jump host to connect to the workload server.
    - In the connection block, the `host` is set to the IP address of the workload server.
    - In the provisioner block, a list of the commands that will be executed on the workload server are listed.

    ```hcl
    resource "null_resource" "application-install" {
      count = var.number_vsi_workload
      connection {
        type         = "ssh"
        user         = "root"
        bastion_host = var.floating_ip_address
        host         = local.workload_ip_list[count.index]
        private_key  = var.ssh_private_key
        agent        = false
        timeout      = "15m"
      }

      provisioner "remote-exec" {
        inline = [
          "apt-get install apache2 -y"
        ]
      }
    }
    ```

To run the Terraform module in your local environment, follow these steps.

> 1.  Clone the repository locally with the following Git command.  Skip this step if done in the [Executing the landing zone with a JSON definition](#/./part2/20-custom-module?id=executing-the-landing-zone-with-a-json-definition):

Reviewer: ```suggestion
1.  Clone the repository locally with the following Git command.  unless you cloned it in the previous step ([Executing the landing zone with a JSON definition](#/./part2/20-custom-module?id=executing-the-landing-zone-with-a-json-definition)):
```
